### ISSUE TITLE
Is it possible to add a <br> after each Feature description line in the report json file ?

### ISSUE THREAD

**Issue by Stephane99** [wswebcreation/multiple-cucumber-html-reporter#204]
**Is your feature request related to a problem? Please describe.**

All my feature files have a description on multi lines.

Eg., the feature contains this description on 2 lines:
```
	This feature contains tests on opening the app from an external link.
	These tests are performed only when the application is already open.
```
But when I generate the html report, the whole description is merged into one line in the report:
```
	This feature contains tests on opening the app from an external link.These tests are performed only when the application is already open.
```

**Describe the solution you'd like**

A workaround is to add a "< br >" html tag before each break line in feature descriptions.
Eg.:
```
	This feature contains tests on opening the app from an external link. <br>
	These tests are performed only when the application is already open <br>
       <br>
       ...
```
But feature descriptions lose clarity.

Is it possible to replace the break lines of the feature description by "< br >" html tags in the report?

**Comment by wswebcreation** [comment]
Good suggestion, anybody wants to pick this up?

**Comment by BaCaRoZzo** [comment]
@Stephane99 FWIW we went the other way around, i.e. we added [`plainDescription`](https://github.com/wswebcreation/multiple-cucumber-html-reporter#plaindescription) to go with custom formatting and then preprocessed the generated `json` files. In that way we kept the `feature` file description simple while adding fancy formatting before the final report is generated. Yes, it's an additional burden but it frees us from being dependent on the lib styling for the description.

Anyway opened a PR to address your request.
